Chinese/ZZB/zzb.md

```diff
@@ -0,0 +1,3 @@
+至尊宝：谁说我斗鸡眼？我只是把视力集中在一点，以改变我以往对事物的看法！
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.